mlflow/ml-package-versions.yml

```diff
@@ -173,6 +173,24 @@ lightgbm:
     run: |
       pytest tests/lightgbm/test_lightgbm_autolog.py --large
 
+catboost:
+  package_info:
+    pip_release: "catboost"
+    install_dev: |
+      temp_dir=$(mktemp -d)
+      git clone --depth 1 https://github.com/catboost/catboost.git $temp_dir
+      cd $temp_dir/catboost/python-package
+      python setup.py bdist_wheel
+      wheel_path=$(find dist -type f -name "*.whl")
+      pip install $wheel_path
+
+  models:
+    minimum: "0.25.1"
```

Comment:
Should we support lower versions ?

Reply:
Good quesion. Let't expand the supported version to [0.2.3.1](https://pypi.org/project/catboost/0.23.1/)

Reply:
0.23.1